### DISCUSSION TITLE
4/15(토) AlertView

### DISCUSSION THREAD

**Discussion by Eunho0922** [Info-iOS/iOStudy#79] [General]
아이폰의 기본 팝업창을 띄울 수 있는 뷰 입니다. AlertView에 대해서 toDoList를 할 때에도 필요할 것으로 느껴져 오늘의 숙제로 내보았습니다.

**Comment by jjunhaa0211** [comment]
알럿 중요하죠~ 내일까지 PR 보내겠습니다 🥹
